examples/vision/video_classification.py

```diff
@@ -37,28 +43,39 @@
 subsampled version of the original UCF101 dataset. You can refer to
 [this notebook](https://colab.research.google.com/github/sayakpaul/Action-Recognition-in-TensorFlow/blob/main/Data_Preparation_UCF101.ipynb)
 to know how the subsampling was done.
+
+#### Note
+
+There was an issue with the link to the dataset, and it was resolved, but at this moment the dataset generation
+notebook is not entirely compatible with this working version of the example, see: https://github.com/keras-team/keras-io/issues/1342
 """
 
 """shell
-wget -q https://git.io/JGc31 -O ucf101_top5.tar.gz
-tar xf ucf101_top5.tar.gz
+wget -q https://zenodo.org/record/7882861/files/ucf101_top10.tar.gz -O ucf101_top10.tar.gz
+tar xf ucf101_top10.tar.gz
 """
 
 """
 ## Setup
 """
 
-from tensorflow_docs.vis import embed
-from tensorflow import keras
-from imutils import paths
-
+import cv2
+import imageio
 import matplotlib.pyplot as plt
-import tensorflow as tf
-import pandas as pd
 import numpy as np
-import imageio
-import cv2
 import os
+import pandas as pd
+import ssl
+import tensorflow as tf
+
+from imutils import paths
+from tensorflow_docs.vis import embed
+from tensorflow import keras
+from tqdm import tqdm
+
+# The SSL part is needed to download Keras' Inception V3 model.
```

Comment:
This shouldn't be needed. @fchollet could you comment?

Reply:
Yes, the comment on that would be helpful. I encountered problem with downloading the Inception V3 model, and the fix with setting SSL parameter resolved the issue. If that could be dealt with some other way, I'd gladly accept that solution. I can provide a log for the error I am getting if there is no SSL param specified.

Thank you for the feedback @sayakpaul !

Reply:
I think this must be an issue with your local SSL config. You're effectively bypassing SSL. However, the target object is hosted on GCS and can be downloaded over SSL using `get_file()` just fine.

`get_file()` will use `urllib`'s `urlretrieve`, which is where your local config error originates.